src/Compilers/CSharp/Test/Emit/CSharpCompilerEmitTest.csproj

```diff
@@ -26,6 +26,9 @@
   </ItemGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(TestIOperationInterface)' == 'true'">
+    <DefineConstants>$(DefineConstants);TEST_IOPERATION_INTERFACE</DefineConstants>
```

Comment:
Feels like we should put this in a common props file somewhere, rather than in a bunch of project files.